disperser/dataapi/queried_operators_handlers.go

```diff
@@ -295,6 +295,19 @@ func (s *server) scanOperatorsHostInfo(ctx context.Context) (*SemverReportRespon
 	if err != nil {
 		return nil, fmt.Errorf("failed to fetch indexed operator info - %s", err)
 	}
+
+	// check operator socket registration against the indexed state
+	for operatorID, operatorInfo := range operators {
+		socket, err := s.chainState.GetOperatorSocket(context.Background(), currentBlock, operatorID)
```

Comment:
will this make extra N RPC calls?

Reply:
Yes, there's no multicast call available from the contract unfortunately

Reply:
@pschork how often is `scanOperatorsHostInfo` called? I believe we have some cronjob?

Reply:
It's called every 12hrs, but the dataAPI is public, so anyone can call it on demand

Reply:
🤔 should we add authentication to this method so not everyone can call it?